src/templates/landing_pages/graduate_program.jsx

```diff
@@ -78,24 +77,25 @@ const GraduateProgram = () => {
                 <div className='gr-row'>
                     <div className='gr-6 gr-12-m'>
                         <div className='box fill-bg-color'>
-                            <p>{('The journey that we are on demands the right company. We seek technologists with a passion for their field and a strong will to resolve complex problems.')}</p>
-                            <p>{('When faced with a problem that you don\'t immediately have the answer to, your determination and curiosity will drive you to a solution. You are a fast learner, and are open to adapting a wide range of technologies. You work well with others, and can collaborate in a fast-paced environment.')}</p>
+                            <p>{('We’re looking for talented graduates who are passionate about software development and can get things done.')}</p>
+                            <p>{('You must possess a natural curiosity and determination to solve complex problems and challenges.')}</p>
+                            <p>{('You must also be a fast learner eager to adopt new tools and technology, and a  dynamic team player who values open and honest communication.')}</p>
                         </div>
                     </div>
                     <div className='separator gr-padding-30 gr-12-m' />
                     <div className='gr-6 gr-12-m'>
-                        <h4 className='center-text'><span className='text-bold'>{('Education and experience requirement')}</span></h4>
+                        <h4 className='center-text'><span className='text-bold'>{('Education and experience requirements')}</span></h4>
                         <div className='inline-flex'>
                             <img className='icon-md margin-30' src={it.url_for('images/graduate_program/education_icon.svg')} />
-                            <p>{('A bachelor\'s, master\'s, or PhD in computer science, IT, mathematics, physics, or engineering.')}</p>
+                            <p>{('A bachelor’s degree, master’s degree, or PhD in computer science, IT, mathematics, physics, or engineering')}</p>
```

Comment:
Actually `\'` is correct. please see [This PR](https://github.com/binary-com/binary-static/pull/5181/files)

Reply:
Sorry, My bad. Fixing it